src/Sdk/AzurePipelines/AzureDevops.cs

```diff
@@ -540,7 +540,6 @@ public class AzureDevops {
                 throw new Exception("Provided undeclared parameters");
             }
```

Comment:
can you provide an example where parameters is referred to as a mapping and not a sequence? I didn't touch this because I couldn't reproduce this scenario.